partcad/tests/unit/test_part_type.py

```
@@ -0,0 +1,73 @@
+    assert root.get_part_type_config("nope") is None
+
+
+@pytest.mark.slow
+def test_wrapper_part_instantiates():
+    """End to end: the wrapper script produces the part's shape (needs the sandbox)."""
+    ctx = pc.Context(EXAMPLE)
+    part = ctx.get_part(":demo")
+    assert part is not None
+    try:
+        wrapped = asyncio.run(part.get_wrapped(ctx))
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)